tests/apps/workflowsapp/Controller.cs

```diff
@@ -15,75 +15,67 @@
 using System.Threading.Tasks;
 using Dapr.Client;
 using System;
+using Dapr.Workflow;
 
 namespace DaprDemoActor
 {
   [ApiController]
   [Route("/")]
   public class Controller : ControllerBase
   {
-    static string httpEndpoint = "http://127.0.0.1:" + Environment.GetEnvironmentVariable("DAPR_HTTP_PORT");
-    static string grpcEndpoint = "http://127.0.0.1:" + Environment.GetEnvironmentVariable("DAPR_GRPC_PORT");
-    public static DaprClient daprClient = new DaprClientBuilder().UseGrpcEndpoint(grpcEndpoint).UseHttpEndpoint(httpEndpoint).Build();
 
+    
     [HttpGet("{workflowComponent}/{instanceID}")]
-    public async Task<ActionResult<string>> GetWorkflow([FromRoute] string instanceID, string workflowComponent)
+    public async Task<ActionResult<string>> GetWorkflow([FromServices]DaprWorkflowClient daprClient,[FromRoute] string instanceID, string workflowComponent)
     {
-      await daprClient.WaitForSidecarAsync();
-      var getResponse = await daprClient.GetWorkflowAsync(instanceID, workflowComponent);
-      return getResponse.RuntimeStatus.ToString();
+      var getResponse = await daprClient.GetWorkflowStateAsync(instanceID);
+      return Enum.GetName(typeof(WorkflowRuntimeStatus), getResponse.RuntimeStatus);
     }
 
     [HttpPost("StartWorkflow/{workflowComponent}/{workflowName}/{instanceID}")]
-    public async Task<ActionResult<string>> StartWorkflow([FromRoute] string instanceID, string workflowName, string workflowComponent)
+    public async Task<ActionResult<string>> StartWorkflow([FromServices]DaprWorkflowClient daprClient,[FromRoute] string instanceID, string workflowName, string workflowComponent)
     {
-      await daprClient.WaitForSidecarAsync();
       var inputItem = "paperclips";
-      var workflowOptions = new Dictionary<string, string>();
-      var startResponse = await daprClient.StartWorkflowAsync(
-              instanceId: instanceID, 
-              workflowComponent: workflowComponent,
-              workflowName: "PlaceOrder",
+      var startResponse = await daprClient.ScheduleNewWorkflowAsync(
+              name:"PlaceOrder",
               input: inputItem,
-              workflowOptions: workflowOptions);
-
-      var getResponse = await daprClient.GetWorkflowAsync(instanceID, workflowComponent);
-
-      return getResponse.InstanceId;
+              instanceId: instanceID);
+      var getResponse = await daprClient.GetWorkflowStateAsync(instanceID);
+      return startResponse;
     }
 
     [HttpPost("PurgeWorkflow/{workflowComponent}/{instanceID}")]
-    public async Task<ActionResult<bool>> PurgeWorkflow([FromRoute] string instanceID, string workflowComponent)
+    public async Task<ActionResult<bool>> PurgeWorkflow([FromServices]DaprWorkflowClient daprClient,[FromRoute] string instanceID, string workflowComponent)
```

Comment:
A few things we need to fix in all these methods:
1. Remove `workflowComponent` from both the route and the parameter list. You may also need to fix the code which invokes these routes to remove their use of `workflowComponent`.
1. Rename `daprClient` to `workflowClient` to avoid confusion with the `DaprClient` class, which we're not using anymore.
2. Please fix the lack of whitespace characters here

Here's an example of what these method signatures should look like:
```csharp
[HttpPost("PurgeWorkflow/{instanceID}")]
public async Task<ActionResult<bool>> PurgeWorkflow([FromServices] DaprWorkflowClient daprClient, [FromRoute] string instanceID)
```

Reply:
Thanks!